feature.env

```diff
@@ -10,7 +10,6 @@ CIAG_INDUCTION_API_URL=http://localhost:9091
 DPS_URL=https://digital-dev.prison.service.justice.gov.uk
 
 STUB_PRISONER_LIST_PAGE_ENABLED=true
-FRONTEND_COMPONENTS_API_FEATURE_TOGGLE_ENABLED=false
```

Comment:
Removed this for the time being as we're not actually using it yet.

Reply:
No, I think we want this back in.
I accept its not used yet .... but I think if we finish the footer story before getting too far ahead ourselves with header, then it will be a bit more cohesive.
Specifically I'm thinking we need to finish RR-143 by doing RR-316 (cypress tests and wiremock stubs) first.

Reply:
Will look at RR-316 now

Reply:
Back in as part of RR-136 👀